pkg/features/kube_features.go

```diff
@@ -1164,6 +1171,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ServiceNodePortStaticSubrange: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.29; remove in 1.31
 
+	SeparateTaintManager: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
nit: move it before `ServiceNodePortStaticSubrange` due to their alphabetical order. (although the items here are not strictly following...)

Reply:
Thanks, fixed in f5a9176